src/frontend/js/components/SearchFilterGroupModal/_styles.scss

```diff
@@ -70,6 +70,7 @@
     background: r-theme-val(search-filters-group-modal, close-background);
```

Comment:
Line 7, instead to use `bottom: 0`, we could use `max-height: calc(100vh - 6rem);` in this way we do not a large blank space when list item is short and viewport height is high.